### PR TITLE
add docs and example on cypress running on sauce

### PR DESCRIPTION
**Pull request by vrunoa**

### CODE REVIEW

**Comment by vrunoa** [comment]
@wswebcreation a quick update on running the same tests in Sauce infrastructure

File: testrunner-toolkit/cypress/README.md [hunk 1]
```diff
@@ -613,3 +613,13 @@ We support Chrome by default, but if you want to run on Firefox please add the f
 capabilities:
   - browserName: firefox
 ```
+
+### Running tests on Sauce Labs cloud
+
```
Comment (wswebcreation) [inline testrunner-toolkit/cypress/README.md:618]:
```suggestion
You can also run Cypress tests on Sauce Labs VM's with the following command:
```

File: testrunner-toolkit/cypress/README.md [hunk 2]
```diff
@@ -613,3 +613,13 @@ We support Chrome by default, but if you want to run on Firefox please add the f
...
+```bash
+saucectl run --test-env sauce
+```
+
```
Comment (wswebcreation) [inline testrunner-toolkit/cypress/README.md:622]:
```suggestion
which will result in this
```

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @vrunoa 

Thanks, should we add the following to pieces of text?
Then it's good to go

**Pull request merged**